google/cloud/aiplatform/initializer.py

```diff
@@ -76,13 +77,21 @@ def project(self) -> str:
         if self._project:
             return self._project
 
-        _, project_id = google.auth.default()
+        try:
+            _, project_id = google.auth.default()
+        except GoogleAuthError:
+            raise ValueError(
```

Comment:
Why do we not simply re-raise the GoogleAuthError?

Reply:
In place of the `ValueError`? Good point, I can do that.